app/mailers/user_notifications.rb

```diff
@@ -53,6 +53,13 @@ def forgot_password(user, opts={})
                 email_token: opts[:email_token])
   end
 
+  def email_login(user, opts={})
+    build_email(user.email,
+                template: "user_notifications.email_login",
+                locale: user_locale(user),
+                email_token: opts[:email_token])
```

Comment:
Not directly related to the PR but I think it is  a good time to refactor some of the methods in this `Mailer`. The only difference between alot of the method is the `template` option. Perhaps we can just extract the duplication into a private method?